pkg/apis/componentconfig/cloudcore/v1alpha1/default.go

```diff
@@ -184,6 +185,7 @@ func NewMinCloudCoreConfig() *CloudCoreConfig {
 				TLSCertFile:       constants.DefaultCertFile,
 				TLSPrivateKeyFile: constants.DefaultKeyFile,
 				AdvertiseAddress:  []string{advertiseAddress.String()},
+				DNSNames:          []string{""},
```

Comment:
No need add it to min and default config.

Reply:
@fisherxu So, we want users to write this field in yaml manually?

Reply:
Sounds good

Reply:
@fisherxu  I will remove it right now.